modules/tensor_mechanics/include/userobjects/TensorMechanicsPlasticJ2.h

```diff
@@ -95,13 +101,12 @@ class TensorMechanicsPlasticJ2 : public TensorMechanicsPlasticModel
   /// max iters for custom return map loop
   unsigned _max_iters;
 
-  /// Whether to use the custom returnMap algorithm
+  /// Whether to use the custom return-map algorithm
   bool _use_custom_returnMap;
```

Comment:
This should probably be const (unless it can be changed after construction). Same with `_use_custom_cto`.